utils/gas-utils/gas_utils.go

```diff
@@ -9,10 +9,13 @@ import (
 
 	"github.com/ava-labs/avalanchego/utils/math"
 	"github.com/ava-labs/subnet-evm/precompile/contracts/warp"
+	teleportermessenger "github.com/ava-labs/teleporter/abi-bindings/go/Teleporter/TeleporterMessenger"
 )
 
 const (
-	ReceiveCrossChainMessageStaticGasCost uint64 = 500_000
+	ReceiveCrossChainMessageStaticGasCost uint64 = 250_000
+	DecodeGasPerByte                      uint64 = 35
+	TeleporterMessageReceiptGas                  = 2500
```

Comment:
```suggestion
	ReceiveCrossChainMessageBaseGasCost uint64 = 250_000
	MarkMessageReceiptGasCost           uint64 = 2500
	DecodeMessageGasCostPerByte         uint64 = 35
```

Nit: I think these names are slightly more consistent/accurate. "Static" was used previously, but technically it's not static so I think it should be renamed.